src/components/commonStyles.css

```diff
@@ -1,4 +1,4 @@
-.changePass-view {
```

Comment:
In the components folder we should only have separate components in directories and this css file doesn't fit here